manifests/init.pp

```diff
@@ -1263,4 +1263,18 @@
       fail("Pam is only supported on RedHat, SuSE, Debian and Solaris osfamilies. Your osfamily is identified as <${::osfamily}>.")
     }
   }
+
+  if $ensure_vas == 'present' and ($::osfamily == 'RedHat' or $::osfamily == 'Debian' or ($::osfamily == 'Suse' and $::lsbmajdistrelease != '9')) {
+    include vas
```

Comment:
vas already includes pam, though it does not appear to reference any of the resources within pam, so this should be safe from preventing circular dependencies. Have you confirmed in your environment by running this code on a new system?